x/bank/keeper/keeper.go

```diff
@@ -320,6 +323,7 @@ func (k BaseKeeper) DelegateCoinsFromAccountToModule(
 	return k.DelegateCoins(ctx, senderAddr, recipientAcc.GetAddress(), amt)
 }
 
+// NOTE(Heimdall-v2): vesting accounts are not used in heimdall
```

Comment:
I might noop this in addition to removing the call to this elsewhere

Reply:
Done [1cbb41b](https://github.com/0xPolygon/cosmos-sdk/pull/5/commits/1cbb41b4cb73390c58ced14d1e5f76288a1802a3)